todo-app/src/Stores/TasksListStore.js

```diff
@@ -0,0 +1,151 @@
+import {action,configure,observable} from 'mobx'
+import moment from 'moment'
+
+import {loadFromLocalStorage,saveToLocalStorage} from '../localStorage'
+import todoList from '../todoList.json'
+
+configure({
+    enforceActions: true
+})
+
+
+
+class TasksListStore {
+
+    @observable
+    todoList = loadFromLocalStorage() || todoList
+
+
+    @observable
+    selectedToEdit = null
+
+    @observable
+    editedTask = ''
+
+    @observable
+    newTask = {
+        description: '',
+        date: '',
+    }
+
+    @action
+    selected = (id) => {
+        const todoItems = this.todoList.map(task => {
+            if (task.id === id) {
+                return {
+                    "id": task.id,
+                    "description": task.description,
+                    "deadline": task.deadline,
+                    "selected": !task.selected,
+                    "isDone": task.isDone,
+                    "timeout": task.timeout
+                }
+            } else {
+                return task
+            }
+        })
+        this.todoList = todoItems
+        saveToLocalStorage(this.todoList)

```

Comment:
Instead of having to put `saveToLocalStorage` at the end of each of your actions, you could use `autorun` to automatically save the store to localStorage each time it changes.